SearchProviders/google_pse.json

```diff
@@ -36,6 +36,7 @@
         "query_mappings": "cx=0c38029ddd002c006,DATE_SORT=sort=date,PAGE=start=RESULT_INDEX,NOT_CHAR=-",
         "result_processors": [
             "MappingResultProcessor",
+            "RequireQueryStringInTitleResultProcessor",
             "DateFinderResultProcessor",
             "CosineRelevancyResultProcessor"
         ],
```

Comment:
@joedinsmoor do we need this change? What was it for again?

Reply:
So this was an addition that Erik made that was causing his issue. Would have to ask him about this

Reply:
@joedinsmoor , is that in the LinkedIn PSE SP within that larger google pse JSON file?  IF so...that's a change we'd like to make to that SP by default (so we'd update preloaded and the db.dist also), but only after the error is sorted out.  So maybe an order of attack question here after all, @dhdnicodemus .

Reply:
> @joedinsmoor , is that in the LinkedIn PSE SP within that larger google pse JSON file? IF so...that's a change we'd like to make to that SP by default (so we'd update preloaded and the db.dist also), but only after the error is sorted out. So maybe an order of attack question here after all, @dhdnicodemus .

That is correct, LinkedIn PSE SP. This isn't what's causing the issue however.

Reply:
Cool, just wanted to make sure that was the one you'd made the change to in this PR.  Once the errors are sorted, we'll make that change in a couple other places too (separate PR).  Thanks for confirming!